ovirtlago/__init__.py

```diff
@@ -210,15 +210,24 @@ def _create_rpm_repository(
         repo_names,
         custom_sources=None,
         projects_list=None,
+        dist_out=None,
     ):
 
         if not projects_list:
             projects_list = PROJECTS_LIST
 
+
         custom_sources = custom_sources or []
 
         rpm_dirs = []
+
+        dist_output = self.paths.internal_repo()
+        if (dist_out):
+            dist_output = dist_out
```

Comment:
it's nicer to do both in the same line, probably without creating an extra var, a 'dummy' example:

`dist_out = dist_out or self.paths.internal_repo()`

The above is readable, but does match empty stings and the like (anything python considers false), if you want to allow them you should use:

`dist_out = dist_out if dist_out is not None else self.paths.internal_repo()`